frontend/src/components/annotation/image_and_video_annotation/annotation_core.vue

```diff
@@ -1898,16 +1898,14 @@ export default Vue.extend({
         this.z_key = false
       })
 
-      this.hotkey_listener.onKeydown({ keys: 'shift', scope }, () => {
+      this.hotkey_listener.onSpecialKeydown({ keys: 'shift', scope }, () => {
```

Comment:
Added `onSpecialKeydown` and `onSpecialKeyup` to bind to individual special hotkeys like shift, control, alt, command. This should be good enough for now. In the future we can make `onKeyup/onKeydown` to handle binding to these individual special keys. Right now `onKeyup/onKeydown` only support special keys like `ctrl` if it's a part of the combination that includes alphanumeric chars (also simbols like ?, !, etc) like `ctrl+c`